Update README.md (#2633)
mikepenz/MaterialDrawer: README.md

// Add for Android-Iconics support
implementation "com.mikepenz:materialdrawer-iconics:${lastestMaterialDrawerRelease}"
```

> Visit [Google's Maven Repository](https://maven.google.com/web/index.html) or [MVN Repository](https://mvnrepository.com/artifact/com.mikepenz/materialdrawer) to find the latest version of the libraries.

Reviewer: ```suggestion
You can find dependency versions and all library releases on [MVN Repository](https://mvnrepository.com/artifact/com.mikepenz/materialdrawer).
```
